schemas/2017-08-01/Microsoft.AnalysisServices.json

```diff
@@ -0,0 +1,219 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2017-08-01/Microsoft.AnalysisServices.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.AnalysisServices",
+  "description": "SQL Analysis Services on Azure",
+  "resourceDefinitions": {
+    "servers": {
```

Comment:
>servers [](start = 5, length = 7)

please add the new resource types to 2015-01-01/deploymentTemplate and 2014-04-01-preview/deploymentTemplate.

Reply:
Already add into templates. Thank you.

Reply:
@YingZhang-MSPBI : I don't see the change.